ls1intum/Artemis
# `Programming exercises`: Fix client validation when creating exercises for certain languages

### Checklist
#### General
#### Client

### Motivation and Context
This PR addresses #4593 and should already been fixed in #4526. Programming exercises with certain languages could not be created as those had no package name set, even if this is irrelevant for creating and cannot be set in the form for these languages.

### Description
The package name is only checked for programming languages that require a package name (Java, Kotlin, Swift).

### Steps for Testing
Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Management
3. Click the button to create a new programming exercise
4. Test all programming languages and verify that all exercises can be created. Validation errors for package names for languages Java, Swift and Kotlin should still be displayed.

### Review Progress
#### Code Review
#### Manual Tests
